ccm_web/server/src/api/api.controller.ts

```diff
@@ -1,8 +1,10 @@
 import { Controller, Get } from '@nestjs/common'
+import { ApiBearerAuth } from '@nestjs/swagger'
 
 import { HelloData, Globals } from './api.interfaces'
 import { APIService } from './api.service'
 
+@ApiBearerAuth()
 @Controller('api')
```

Comment:
Should the ConsumerTest component be using this?

Reply:
I'm not sure what you mean, can you explain? Do you think `api/hello` shouldn't be behind the authentication? Since we're installing `ltijs` as global middleware, I'm not sure we have that level of control. Also, this decorator is just telling the Swagger UI what authentication is needed to use the endpoints, it doesn't actually control it.

Reply:
I was thinking that all API calls would go through the API Controller.  That's not the case?

I didn't think about it deeply, that is just how I have  done things in the past in my own code, where nobody is working with me to tell me I'm doing it wrong :-D

Reply:
Hm, well, the `ConsumerTest` API calls the `hello` endpoint that is defined here, right? So it is going through this.

Reply:
You're right, the way I was searching the code I missed the `fetch('/api/hello', params)`